app/workers/my_anime_list_sync_worker.rb

```diff
@@ -5,16 +5,20 @@ class MyAnimeListSyncWorker
   def perform(data)
     # TODO: make a report per day for what is happening (syncing)
     case data['method']
-    when 'delete'
+    when :delete
```

Comment:
Sidekiq parameters get turned into JSON and back, so you won't get a symbol of `:delete`, you'll probably get a string of `'delete'`